sklearn/linear_model/_least_angle.py

```diff
@@ -1111,17 +1111,17 @@ class LassoLars(Lars):
 
     Attributes
     ----------
-    alphas_ : array-like of shape (n_alphas + 1,) or list of thereof of shape \
+    alphas_ : array-like of shape (n_alphas + 1,) or list thereof of shape \
             (n_targets,)
```

Comment:
I still find this difficult to parse. Because it uses "shape" that is usually associated with an ndarray, but in this case it means the length of the list. The other way to parse this is:

> array-like of shape (n_alphas + 1,) or a list of array-like of shape (n_targets,)

Maybe:

> array-like of shape (n_alphas + 1,) or a list of length n_targets thereof

I'm still unsure about this.

Reply:
I agree this is hard to parse. I think that in this case, we can specify the length of the list in the description below instead:

```
alphas_ : array-like of shape (n_alphas + 1,) or list thereof
	Blah blah blah...
	If this is a list, its length is `n_targets`.
```

Reply:
I like the solution of @NicolasHug